x/cert/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,210 @@
+package keeper_test
+
+import (
+	"github.com/certikfoundation/shentu/v2/x/cert/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+)
+
+func (suite *KeeperTestSuite) TestQueryCertifier() {
+	ctx, queryClient := suite.ctx, suite.queryClient
+
+	tests := []struct {
+		address    string
+		alias      string
+		shouldPass bool
+	}{
+		{
+			address:    "",
+			alias:      "",
+			shouldPass: false,
+		},
+		{
+			address:    suite.address[0].String(),
+			alias:      "",
+			shouldPass: true,
+		},
+		{
+			address:    "",
+			alias:      "address1",
+			shouldPass: true,
+		},
+		{
+			address:    "",
+			alias:      "invalid",
+			shouldPass: false,
+		},
+	}
+
+	for _, tc := range tests {
+		queryResponse, err := queryClient.Certifier(ctx.Context(), &types.QueryCertifierRequest{Address: tc.address, Alias: tc.alias})
+		if tc.shouldPass {
+			suite.Require().NoError(err)
+			if tc.address != "" {
+				suite.Equal(tc.address, queryResponse.Certifier.Address)
+			} else {
+				suite.Equal(tc.alias, queryResponse.Certifier.Alias)
+			}
+		} else {
+			suite.Require().Error(err)
+		}
+	}
+}
+
+func (suite *KeeperTestSuite) TestQueryCertifiers() {
+	ctx, queryClient := suite.ctx, suite.queryClient
+
+	// TODO: Need clarification
```

Comment:
Thanks for review @hacheigriega!
Since I passed `nil` in `queryClient.Certifiers(ctx.Context(), nil)`, I think it should return an error, but it isn't.

Reply:
I think gRPC takes care of nil requests.

Reply:
Sending an empty request like `&types.QueryCertifierRequest{}` results in a `nil` response.

Reply:
Yes, but for `&types.QueryCertifiersRequest{}` (query certifier**s**) it passes without any error. I think we should remove the check `req==nil` for this case since it doesn't require any argument.